### PR TITLE
Check installable as a wheel

### PR DESCRIPTION
**Pull request by jrapin**
This removes the upload on the test pypi server and just checks that the wheel can be installed

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -41,7 +41,8 @@ mypy:
 
 pylint:
 	$(BIN)pylint --version
-	$(BIN)pylint $(CODE)
+	$(BIN)pylint $(CODE) --disable=unspecified-encoding
```
Comment (gwenzek) [inline Makefile:44]: the pylint config is here: https://github.com/facebookincubator/submitit/blob/dac5c6b76a81401aa7d44a32357872ebf92b968a/pyproject.toml#L21

but maybe it's a valid concern ? Not everyone is using utf8 as default enconding

**Review by gwenzek — CHANGES_REQUESTED** [review]
thanks for the improvements !

could you move the pylint flag to pyproject.toml ?

File: Makefile (hunk 1 above)
Comment (jrapin) [reply Makefile:44]: it could be valid for non-test stuff (tests are most of the cases), then again... not so sure :s is it better to force utf8 or not...

Comment (jrapin) [reply Makefile:44]: so dunno, i'd say no until someone complains

Comment (gwenzek) [reply Makefile]: ok, good for me.

**Pull request merged**